tests/test_multidict.py

```diff
@@ -1189,3 +1191,45 @@ def test_items_case_insensitive_isdisjoint(
     ) -> None:
         d = cls([("KEY", "one")])
         assert d.items().isdisjoint(arg) == expected
+
+
+@pytest.mark.xfail(reason="issue #1111", strict=True)
+def test_convert_multidict_to_cimultidict() -> None:
+    """Test conversion from MultiDict to CIMultiDict."""
+    start_as_md = MultiDict([("KEY", "value1"), ("key2", "value2")])
+    assert start_as_md.get("KEY") == "value1"
+    assert start_as_md["KEY"] == "value1"
+    assert start_as_md.get("key2") == "value2"
+    assert start_as_md["key2"] == "value2"
+    start_as_cimd = CIMultiDict([("KEY", "value1"), ("key2", "value2")])
+    assert start_as_cimd.get("key") == "value1"
+    assert start_as_cimd["key"] == "value1"
+    assert start_as_cimd.get("key2") == "value2"
+    assert start_as_cimd["key2"] == "value2"
+    converted_to_ci = CIMultiDict(start_as_md)
+    assert converted_to_ci.get("key") == "value1"
+    assert converted_to_ci["key"] == "value1"
+    assert converted_to_ci.get("key2") == "value2"
+    assert converted_to_ci["key2"] == "value2"
+
+
+@pytest.mark.xfail(reason="issue #1111", strict=True)
+def test_pure_python_convert_multidict_to_cimultidict_eq() -> None:
```

Comment:
Shouldn't these be run on both implementations with a fixture?

Reply:
It doesn't fail on the C version

Reply:
But it may gain the same bug over time. I think the value of using the parametrized fixtures is in making sure that the behavior is the same across implementations. And this would help making sure that any regression fixes don't contribute to diverging behavior.

Reply:
It makes sense to change it now that it's a positive test instead of an xfail

Reply:
Adjusted to test all versions